doc/modules/model_persistence.rst

```diff
@@ -63,9 +63,11 @@ Security & maintainability limitations
 pickle (and joblib by extension), has some issues regarding maintainability
 and security. Because of this,
 
-* Never unpickle untrusted data
-* Models saved in one version of scikit-learn might not load in another
-  version.
+* Never unpickle untrusted data as it could lead to malicious code being 
+  executed upon loading.
+* While models saved using one version of scikit-learn will likely load in 
+  other versions, it should be kept in mind that operations performed on them 
+  could give different and unexpected results.
```

Comment:
Maybe you should be a bit more explicit that this is not supported and that it is not advised to work like this.